artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -1733,8 +1734,16 @@ public void acknowledge(final Transaction tx, final MessageReference ref, final
          if (AuditLogger.isMessageLoggingEnabled()) {
             // it's possible for the consumer to be null (e.g. acking the message administratively)
             final ServerSession session = consumer != null ? server.getSessionByID(consumer.getSessionID()) : null;
-            final Subject subject = session == null ? null : session.getRemotingConnection().getSubject();
-            final String remoteAddress = session == null ? null : session.getRemotingConnection().getRemoteAddress();
+            final Subject subject;
+            final String remoteAddress;
+            if (session == null) {
+               subject = new Subject();
+               subject.getPrincipals().add(new UserPrincipal("management"));
+               remoteAddress = "internal";
```

Comment:
If this is arbitrary I'd possibly consider something other than 'management', which often seems more associated with explicit user operations. Perhaps 'system', or 'broker' ?